### ISSUE TITLE
[REF] Rename 'pos' to something more meaningful

### ISSUE THREAD

**Issue by christophKaiser** [tumcms/Open-Infra-Platform#533]
The parameter name `pos` is used in many functions of the geometry converter. It describes the transformation of the geometry coordinates to place an object in its place. A [change request](https://github.com/tumcms/Open-Infra-Platform/pull/462#discussion_r805921861) brought up that the parameter name does not really fit.

In the discussion and in a dev-meeting, the suggestion `objectPlacement` was decided to be a more meaningful name.

**File which is being refactored**
Many files in the namespace IfcGeometryConverter.

**Functions**
Basically, all functions in the IfcGeometryConverter could be affected and should be checked (at least with the auto-search/-replacement tool).
